classes/class.memberorder.php

```diff
@@ -1184,4 +1769,68 @@ function get_test_order() {
 
 			return apply_filters( 'pmpro_test_order_data', $this );
 		}
-	} // End of Class
\ No newline at end of file
+<<<<<<< HEAD
+	}
+
+	/**
+	 * Delete an order and associated data.
+	 */
+	function deleteMe() {
+		if ( empty( $this->id ) ) {
+			return false;
+		}
+
+		global $wpdb;
+		$this->sqlQuery = "DELETE FROM $wpdb->pmpro_membership_orders WHERE id = '" . $this->id . "' LIMIT 1";
+		if ( $wpdb->query( $this->sqlQuery ) !== false ) {
+			do_action( 'pmpro_delete_order', $this->id, $this );
+			return true;
+		} else {
+			return false;
+		}
+	}
+
+	/*
+	* Generates a test order on the fly for orders.
+	*/
+	function get_test_order() {
+		global $current_user;
+
+		$all_levels = pmpro_getAllLevels();
+
+		if ( ! empty( $all_levels ) ) {
+			$first_level          = array_shift( $all_levels );
+			$this->membership_id  = $first_level->id;
+			$this->InitialPayment = $first_level->initial_payment;
+		} else {
+			$this->membership_id  = 1;
+			$this->InitialPayment = 1;
+		}
+		$this->user_id             = $current_user->ID;
+		$this->cardtype            = 'Visa';
+		$this->accountnumber       = '4111111111111111';
+		$this->expirationmonth     = date( 'm', current_time( 'timestamp' ) );
+		$this->expirationyear      = ( intval( date( 'Y', current_time( 'timestamp' ) ) ) + 1 );
+		$this->ExpirationDate      = $this->expirationmonth . $this->expirationyear;
+		$this->CVV2                = '123';
+		$this->FirstName           = 'Jane';
+		$this->LastName            = 'Doe';
+		$this->Address1            = '123 Street';
+		$this->billing             = new stdClass();
+		$this->billing->name       = 'Jane Doe';
+		$this->billing->street     = '123 Street';
+		$this->billing->city       = 'City';
+		$this->billing->state      = 'ST';
+		$this->billing->country    = 'US';
+		$this->billing->zip        = '12345';
+		$this->billing->phone      = '5558675309';
+		$this->gateway_environment = 'sandbox';
+		$this->timestamp           = time();
+		$this->notes               = __( 'This is a test order used with the PMPro Email Templates addon.', 'paid-memberships-pro' );
+
+		return apply_filters( 'pmpro_test_order_data', $this );
+	}
+} // End of Class
+=======
+	} // End of Class
+>>>>>>> parent of 9b238692 (WPCS Formatting)
```

Comment:
Probably from bad merge.
```suggestion
} // End of Class
```